lib/pymedphys/_utilities/test.py

```diff
@@ -74,15 +74,19 @@ def get_executable_even_when_embedded():
         except FileNotFoundError:
             pass
 
-    exe = str(pathlib.Path(np.__file__).parents[4].joinpath("bin", "python"))
+    exe = str(
+        pathlib.Path(np.__file__).parents[4].joinpath("bin", "python")
+    )  # pylint: disable=no-member
```

Comment:
This need for no-member has me confused...

Reply:
One possibility is that `pylint` somehow recognises that `pathlib.Path(np.__file__).parents[4]` could return an Exception (IndexError) and so `joinpath()` would not be a "member". Dunno.